mesonbuild/backend/ninjabackend.py

```diff
@@ -1709,17 +1709,23 @@ def generate_rust_target(self, target: build.BuildTarget) -> None:
                 _ods, main_rust_file = self.__generate_compile_structure(target)
                 orderdeps.extend(_ods)
             else:
+                # The only way to get here is to have a only root files at are
```

Comment:
Seems like there's a couple typos here. At least "at" -> "that", and "a only root files" is probably not supposed to have the leading "a"?

Reply:
I reworded it